docs/shared/ioda.yaml

```diff
@@ -12,7 +12,6 @@ ioda:
       - module load some-module
       - module load ioda-module
     executable: /path/to/a/ioda/exe
-    mpicmd: time
```

Comment:
Not allowed.

Reply:
This is done for every command, right?

Reply:
Only this one needed to change, as `ioda` uses the `execution-serial` schema. Other drivers use the full `execution` schema that _does_ allow `mpicmd`.